### PR TITLE
🏠 🍴 ➕ 💧 💻  Finish It. The Icons. 🏠 🍴 ➕ 💧 💻 

### PR DESCRIPTION
**Pull request by thebritican**
Icons are now present (correctly) through the entire app. This may need a styling pass later on, but all the logic is done.

### Locations page
image

### Location page

image

selfie-0
[_GitHub Selfies_](https://github.com/thieman/github-selfies/)

### CODE REVIEW

File: components/Location/Location.css [hunk 1]
```diff
@@ -124,6 +124,10 @@
   font-size: 14px;
 }
 
+.categoryIcons {
+  display: flex;
```
Comment (roseleaf) [inline components/Location/Location.css]: 💪

File: components/Location/Location.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react'
 import s from './Location.css'
 import icons from '../../icons/css/icons.css'
-
 import GoogleMap from '../GoogleMap'
+import { capitalize, relevantTaxonomies, getIcon } from '../../lib/categories'
```
Comment (craig-day) [inline components/Location/Location.js]: `capitalize` is not a `categories` thing.... get it outta that lib!

File: pages/locations/index.js [hunk 3]
```diff
@@ -74,13 +73,17 @@ export default class LocationsPage extends Component {
 
   render() {
     const { locations } = this.state
+    const filteredLocations = locations.filter(loc => (
+      loc.services &&
+      loc.services.filter(service => service.taxonomy === 'housing'))
```
Comment (craig-day) [inline pages/locations/index.js]: why the hardcode to `housing` here?

**Comment by thebritican** [comment]
This also makes the default filter to "housing". We'll change this when we have another PR for query param based routing

File: components/Location/Location.js (hunk 2 above)
Comment (thebritican) [inline components/Location/Location.js]: going to call yours once it's merged 😉

File: pages/locations/index.js (hunk 3 above)
Comment (thebritican) [inline pages/locations/index.js]: See below

File: components/Location/Location.js (hunk 2 above)
Comment (craig-day) [inline components/Location/Location.js]: well then go review mine! :trollface:

**Comment by craig-day** [comment]
:+1:

**Pull request merged**